pages/apim/3.x/user-guide/publisher/plans-subscriptions.adoc

```diff
@@ -62,6 +62,8 @@ This plan ensures that API keys are valid, have not been revoked or expired, and
 NOTE: The API key plan represents the simplest security mechanism and acts more as a unique identifier than a security token since the API key can be easily found in app code.
 For stronger security please see OAuth 2.0 and JWT plans.
 
+NOTE: With the API key plan, it is possible to allow custom API key. The publisher is now able to silently migrate to Gravitee.io without changing consumers API key.
```

Comment:
Rewrite as 'You can specify a custom API key for an API key plan. This is particularly useful when you want to silently migrate to APIM and have a pre-defined API key. '

Also specify other useful information such as you can specify or generate an API key [I think - from what I understand of the functionality] and the API key must have more than 8 characters, less than 64 characters and be URL compliant (for example, not include escape characters) [link to more info on restrictions if possible].

Also pls add a screenshot.